skyportal/handlers/source.py

```diff
@@ -237,3 +237,34 @@ def post(self):
             info['sourceNumberingStart'] = 0
 
         return self.success(data=info)
+
+
+class SourcePhotometryHandler(BaseHandler):
+    @auth_or_token
+    def get(self, source_id):
+        """
+        ---
+        description: Retrieve a source's photometry
+        parameters:
+        - in: path
+          name: source_id
+          required: false
+          schema:
+            type: integer
+        responses:
+          200:
+            content:
+              application/json:
+                schema: ArrayOfPhotometrys
+          400:
+            content:
+              application/json:
+                schema: Error
+        """
+        info = {}
+        source = Source.get_if_owned_by(source_id, self.current_user)
+        if not source:
+            return self.error('Invalid source ID, or inadequate permissions to '
+                              'access source.')
+        info['photometry'] = source.photometry
```

Comment:
No big deal, but could probably just be `info = {'photometry': source.photometry}` instead of instantiating dict separately, or could even just put this inside the return statement.

Reply:
Done!